_includes/tutorials/dynamic-output-topic/kstreams/code/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: '2'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.4.0
```

Comment:
Let's bump the images to 5.4.1 here and below.

Reply:
ack